tests/components/application-connector/README.md

```diff
@@ -2,65 +2,172 @@
+
+``` sh
+make test-gateway
+```
+
+By default tests clean up after themselves, removing all created resources
+and `test` Namespace.
+
+> **CAUTION:** This might override or remove existing resources, 
+> if names of already existing resources collide with names used by tests
+
+## Debugging
 
-   <div tabs name="Kyma flavor" group="minimal-kyma-installation">
-      <details open>
-      <summary label="OS">
-      Kyma OS
-      </summary>
+### Running locally
 
-   ```bash
-   kyma deploy --components-file ./resources/installation-config/mini-kyma-os.yaml
+The Test Job must run on a cluster, because of the way it accesses Application CRs.
+Application Gateway and Mock Application can both be run locally.
+
+To run Mock Application locally:
+1. Change all `targetUrl` in [Application CRs](./resources/charts/gateway-test/templates/applications/)
+   to reflect new app URL. For example `http://localhost:8081/v1/api/unsecure/ok`
+1. Change all `centralGatewayUrl` to reflect new Application Gateway URL. 
+   For example `http://localhost:8080/positive-authorisation/unsecure-always-ok`
+1. Deploy all resources (you can omit test Job and Central Gateway, but it's easier to just let them fail)
+   on the cluster
+1. Build the Mock Application:
+   
+   <div tabs name="Mock App Build Flavor" group="mock-app-flavor">
+   <details open>
+   <summary label="dockerized">
+   Docker
+   </summary>
+
+   ```shell
+   export DOCKER_TAG="local"
+   export DOCKER_PUSH_REPOSITORY="{Docker username}"
+   make image-mock-app
    ```
-      </details>
-      <details>
-      <summary label="SKR">
-      SKR
-      </summary>
-
-   ```bash
-   kyma deploy --components-file ./resources/installation-config/mini-kyma-skr.yaml 
+
+   </details>
+   <details>
+   <summary label="local">
+   Local
+   </summary>
+
+   Change hardcoded application port in [config.go](./tools/external-api-mock-app/config.go), then
+   ```shell
+   go build ./tools/external-api-mock-app/
    ```
-      </details>
+   </details>
    </div>
+1. Run the Mock Application:
+   
+   <div tabs name="Mock App Run Flavor" group="mock-app-flavor">
+   <details open>
+   <summary label="dockerized">
+   Docker
+   </summary>
+
+   ```shell
+   docker run -p 8180:8080 -p 8190:8090 -v "$PWD/k8s/gateway-test/certs:/etc/secret-volume:ro" "$DOCKER_PUSH_REPOSITORY/mock-app:$DOCKER_TAG"
+   ```
 
-3. Run the tests:
+   </details>
+   <details>
+   <summary label="local">
+   Local
+   </summary>
 
-   ```bash
-   make test-gateway
+   ```shell
+   ./external-api-mock-app
    ```
+   > **CAUTION:** Certificates won't work, unless you copy them from `./k8s/gateway-test/certs` to `/etc/secret-volume`
+
+   </details>
+   </div>
+1. Run [Central Application Gateway](https://github.com/kyma-project/kyma/tree/main/components/central-application-gateway),
+   with `-kubeConfig {path_to_kubeconfig.yaml}` parameter.
+
+You can now send requests to the Application Gateway and debug its behaviour locally.
```

Comment:
```suggestion
You can now send requests to Application Gateway, and debug its behavior locally.
```
Nice that you included the result :)

- We don't use the definite article with the names of our components. 
- `behaviour` is the BrE spelling, we use AmE (`behavior`) in our documentation. See the [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#terminology) for reference.